Workstream Policy.md

```diff
@@ -303,6 +303,16 @@ In the text above, "Living Standard" must link to the corresponding
 "exclusion notices" must link to a location determined by the [Steering
 Group] where patent exclusion notices are posted.
 
+### Notice for Supporting Resources
```

Comment:
Lowercase "supporting resources" since it's not a defined term

Reply:
Hmm, I am fine with the title case on this.

Reply:
I think we used sentence case instead of title case for other headers, when not using defined terms. So I think @domenic's suggestion is correct.